python/device/device.py

```diff
@@ -145,7 +144,7 @@ def __init__(self):
         self.received_pingback_list = []
         self.new_pingback_event = threading.Event()
         # for telemetry
-        self.telemetry_queue = queue.Queue()
+        self.outgoing_send_message_queue = queue.Queue()
```

Comment:
interesting that the previous change as the order of send_outgoing, but this one is outgoing_send.

Reply:
I'll take a look at that in my next PR.  That is interesting...consistency is very important in my world and I think I had a reason for this, but I'll make sure.